R/data_reshape.R

```diff
@@ -165,7 +165,7 @@ data_to_long <- function(data,
 
   # Cleaning --------------------------
   # Sort the dataframe (to match pivot_longer's output)
-  long <- long[do.call(order, long[, c("_Row", names_to_2)]), ]
+  long <- data_arrange(long, c("_Row", names_to_2))
```

Comment:
w/o this change, it's even a bit faster, or was there a particular reason for calling `data_arrange()`?

Reply:
ah I thought it was equivalent in speed but more readable, but I should revert it if do.call is faster

Reply:
My benchmark showed 1.62 vs 1.52 seconds, not much, but maybe still?

Reply:
Yeah on my side I win almost 0.5 sec (it changes at each bench::mark run but still do.call is faster. I'll revert the change

Reply:
I think the minimal overhead is regex usage:

```r
  desc <- select[grepl("^-", select)]
  desc <- gsub("^-", "", desc)
  select <- gsub("^-", "", select)
```

Reply:
using `fixed = TRUE` barely reduces the time for `data_arrange`, the overhead probably comes from all the checks in `data_arrange`